### ISSUE TITLE
Improvement: set multiple timers?

### ISSUE THREAD

**Issue by ilippert** [MajortomVR/simple-timer-extension#2]
Wow, this timer looks neat and nice.
As per the extensions site 

>  Simple but effective! An option to set multiple timers would be great.

I agree very much with that :)
and to make the timers distinct, naming them would make sense

**Comment by MajortomVR** [comment]
Thanks for the suggestions!

Do you mean to have multiple timers running at the same time, 
or the option to choose from multiple custom timers from the menu?

**Comment by ilippert** [comment]
> Do you mean to have multiple timers running at the same time,

yes, this is the idea. e.g. a timer for tea, and a timer for a meeting break
